backend/src/controllers/serviceControllers.js

```diff
@@ -0,0 +1,38 @@
+import githubService from '../thirdParties/github/githubService';
+
+// we are supporting one service now, but in
+// the future we can add more services.
+const getAppropriateService = serviceName => {
+  switch (serviceName) {
```

Comment:
Still it's the same...no matter what the condition is, you are returning githubService. Which condition are you testing